grow/pods/podcache.py

```diff
@@ -41,12 +55,59 @@ def document_cache(self):
         """Cache for specific document properties."""
         return self._document_cache
 
+    @property
+    def object_cache(self):
+        """Global object cache."""
+        return self.get_object_cache(self.KEY_GLOBAL)
+
+    def create_object_cache(self, key, write_to_file=False, can_reset=True, values=None):
```

Comment:
I was going for more of a canonical way of caching. Then we can document and it would allow the caches to be easily shared between extensions if they understood the cache structure that the other extension is using.